core/io/config_file.cpp

```diff
@@ -123,6 +123,11 @@ void ConfigFile::erase_section(const String &p_section) {
 	values.erase(p_section);
 }
 
+void ConfigFile::erase_section_key(const String &p_section, const String &p_key) {
+
+	values[p_section].erase(p_key);
```

Comment:
This will crash if `p_section` is not in the `values`.

You should add a check like in other `_section_key` methods right above. E.g.:
```
ERR_FAIL_COND_MSG(!values.has(p_section), "Cannot erase key from nonexistent section '" + p_section + "'.");
```